README.md

```diff
@@ -18,10 +18,16 @@
 
 To get started with RunPod File Uploader, execute the following command:
 
+#### Installation
```

Comment:
Don't think this is needed because its already under the "How to Install section".